torchao/experimental/ops/mps/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+python ../../kernels/mps/codegen/gen_metal_shader_lib.py
+
+export CMAKE_PREFIX_PATH=$(python -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())')
+echo "CMAKE_PREFIX_PATH: ${CMAKE_PREFIX_PATH}"
+export CMAKE_OUT=cmake-out
+
+cmake -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH} \
+    -DCMAKE_INSTALL_PREFIX=${CMAKE_OUT} \
+    -S . \
+    -B ${CMAKE_OUT}
+cmake --build  ${CMAKE_OUT} -j 16 --target install --config Release
+
+rm ../../kernels/mps/src/metal_shader_lib.h
```

Comment:
wait what is this?

Reply:
after I am done building and installing, I don't need the generated metal_shader_lib.h anymore.

Reply:
ok then generated artifacts should be stored in build directory as temporary artifacts or somewhere else. Adding in the repo folder and removing it can leave the repo in dirty state if build fails. Also generally clean